lib/repositories/user/source_user_local.dart

```diff
@@ -22,6 +22,9 @@ abstract class LocalUserSource {
   /// If no [User] or [Wallet] have been saved yet, returns `null`.
   Future<MooncakeAccount> getAccount();
 
+  /// Returns a list of accounts currently logged in
```

Comment:
It's probably better to have this return a list of accounts that are stored locally instead